superset/utils/core.py

```diff
@@ -650,11 +671,13 @@ def get_email_address_list(address_string):
     if isinstance(address_string, basestring):
         if ',' in address_string:
             address_string = address_string.split(',')
+        elif '\n' in address_string:
```

Comment:
NIT: unrelated to your PR, but I'm realizing that only a single breaking scheme is supported here. I think we could just do `address_string.split(',\n; ')` in a single pass here